lib/unit/api_resources/payment_resource.rb

```diff
@@ -10,7 +10,7 @@ module Resource
     class PaymentResource < Unit::Resource::BaseResource
```

Comment:
Received payment is not Payment, its is own resource.
Please split all the received-payments ones into their own files/packages. (payment is when our user sends money, received-payment is when someone sends him money)